index.html

```diff
@@ -1,13 +1,36 @@
 <!doctype html>
 <html lang="en">
-  <head>
+
+<head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite App</title>
-  </head>
-  <body>
+</head>
+
+<body>
     <div id="app"></div>
     <script type="module" src="/main.js"></script>
-  </body>
+
+    <div id="buttons" align="center">
+        <button>
+            <tooltip placement="topLeft" trigger="hover">HI :)</tooltip>
```

Comment:
First note - `<tooltip>` element.
There is not element in a specification about this element, but it is valid syntax, because we have [Web Components](https://developer.mozilla.org/en-US/docs/Web/API/Web_components).

Declaring a web-component can look like that:
https://github.com/mdn/web-components-examples/blob/main/popup-info-box-web-component/main.js
You create a special class and then define this as HTML-Tag

Here is the example how it can be done:
https://opensource.adobe.com/spectrum-web-components/components/overlay-trigger/#%22hover%22-content-only

It is quite complex examples to implement, but I hope it could be interesting to investigate